src/main/java/edu/kis/vh/nursery/factory/Rhymersfactory.java

```diff
@@ -1,15 +1,15 @@
 package edu.kis.vh.nursery.factory;
 
-import edu.kis.vh.nursery.defaultCountingOutRhymer;
-
-	public interface Rhymersfactory {
-	
-		public defaultCountingOutRhymer GetStandardRhymer();
-		
-		public defaultCountingOutRhymer GetFalseRhymer();
-		
-		public defaultCountingOutRhymer GetFIFORhymer();
-		
-		public defaultCountingOutRhymer GetHanoiRhymer();
-		
-	}
+import edu.kis.vh.nursery.DefaultCountingOutRhymer;
+
+public interface Rhymersfactory {
+
+    DefaultCountingOutRhymer getStandardRhymer();
```

Comment:
Original project formatting convention used tabs (should not be changed to spaces). It makes tracking actual changes more difficult.